_pages/how-we-work/tools/google-meet.md

```diff
@@ -42,6 +42,16 @@ If the audio for Hangouts has stopped working, you can either reset your compute
 4. If it asks for your password, type your computer log-in password and then hit enter
 5. Audio should now work, but if it doesn't, try re-starting your computer
 
+### Presenting a deck in hangouts
+If you're doing a virtual presentation it can be a bit tricky, so here are some tips to both see what you are presenting and who you are presenting to.
+
+1. Open up in your browser your slides presentation. (Window a)
+2. Click on present with presenter view and a new window will pop up for you to see a thumbnail of your slides and your notes. (Window b)
+2. Fire up your meet meeting in a browser new window. (Window c)
```

Comment:
Capitalizing "meet" will make it clearer that this is referring to a specific product (meet meeting is a weird phrase otherwise).

Reply:
@lauraGgit @coreycaitlin what else needs to be done before this can be merged?

Reply:
We can go ahead and merge, and I can tidy the content stuff in a separate PR!